samples/ravendb/simple/sample.md

```diff
@@ -29,20 +29,13 @@ This sample shows a simple Client + Server scenario.
 
 ### Raven Config
 
-Configure the endpoint to use RavenDB persistence.
+Configure the endpoint to use RavenDB persistence. The Url/Port may need to be changed to match the database in use.
```

Comment:
Why are "Url" and "Port" capitalized?

Reply:
Url will be capitalized in the snippet, though that technically contains the port too. Perhaps just "The Url may be..." would be better.

Reply:
👍 For "The Url" since that includes the port.